centreon/www/front_src/src/Resources/Actions/Visualization/Visualization.tsx

```diff
@@ -1,20 +1,20 @@
 import { useTranslation } from 'react-i18next';
 
-import { Box, Typography } from '@mui/material';
+import { Grid, Typography } from '@mui/material';
 
-import viewByServiceActive from '../../icons/view_service_actif.svg';
-import viewByServiceInactive from '../../icons/view_service_inactif.svg';
+import viewByAllActive from '../../icons/view_all_actif.svg';
+import viewByAllInactive from '../../icons/view_all_inactif.svg';
 import viewByHostActive from '../../icons/view_host_actif.svg';
 import viewByHostInactive from '../../icons/view_host_inactif.svg';
-import viewByAllInactive from '../../icons/view_all_inactif.svg';
-import viewByAllActive from '../../icons/view_all_actif.svg';
+import viewByServiceActive from '../../icons/view_service_actif.svg';
+import viewByServiceInactive from '../../icons/view_service_inactif.svg';
```

Comment:
FYI, this will be refactored by providing those icons from Centreon UI using the SvgIcon component from MUI.
Thus no action is needed here

Reply:
that was just an eslint  reorder ,i didn t use those svgs